SAMv1.tex

```diff
@@ -217,7 +217,11 @@ \subsection{The header section}
 \emph{Regular expression}: \emph{name}{\tt (,}\emph{name}{\tt )*}
 where \emph{name} is {\tt [0-9A-Za-z][0-9A-Za-z*+.@\_|-]*}\\\cline{2-3}
   & {\tt AS} & Genome assembly identifier. \\\cline{2-3}
-  & {\tt M5} & MD5 checksum of the sequence in the uppercase, excluding spaces but including pads (as `*'s).\\\cline{2-3}
+  & {\tt M5} & MD5 checksum of the sequence.
+The sequence must be in the 7-bit US-ASCII character set, with pads represented as `*'s if present.
+Before calculating the digest, all characters outside the inclusive range 33 (`!') to 126 (`\~{}')  are removed, and all lower-case characters are converted to upper-case\footnote{Equivalent to calling \href{http://pubs.opengroup.org/onlinepubs/9699919799/functions/toupper.html}{\sl toupper()} on the characters in the \href{http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1\_chap07.html\#tag\_07\_02}{\sl POSIX locale}}.
+The MD5 digest\footnote{See \href{https://tools.ietf.org/html/rfc1321}{\sl RFC 1321}} is calculated and presented as a 32 character lower-case hexadecimal number.
```

Comment:
Should we specify that the result should not be preceded by `0x`? 

How about adding an illuminating example? 
>For example, if the content of the reference sequence is:
"ACGT ACGT ACGT
acgt acgt acgt
... 12345 !!!
"

the resulting md5 is that of the string "ACGTACGTACGTACGTACGTACGT...12345!!!", namely 
>M5:cc14900b11b876f79231ce3a5860796b